libavcodec/x86/vvcdsp_init.c

```diff
@@ -137,6 +139,29 @@ static void alf_classify_10_avx2(int *class_idx, int *transpose_idx,
         c->alf.classify = alf_classify_##depth##_avx2;                          \
     } while (0)
 
+void ff_vvc_put_vvc_luma_hv_10_avx512icl(int16_t *dst, const uint8_t *_src, const ptrdiff_t _src_stride,
+    const int height, const intptr_t mx, const intptr_t my, const int width,
+    const int hf_idx, const int vf_idx);
+
+void (*p_vvc_put_vvc_luma_hv_10_tt)(int16_t *dst, const uint8_t *_src, const ptrdiff_t _src_stride,
+    const int height, const intptr_t mx, const intptr_t my, const int width,
+    const int hf_idx, const int vf_idx);
+
+static void vvc_put_vvc_luma_hv_10_wrap(int16_t *dst, const uint8_t *_src, const ptrdiff_t _src_stride,
```

Comment:
Is it possible to write a 16bpc function for 10, 12 bits?
this may save some binary size like https://github.com/ffvvc/FFmpeg/issues/14 mentioned

Reply:
Sure. We can change the hv_10 to hv_16. These functions should already support 10, 12, and 16bits samples.